### ISSUE TITLE
Shiny app won't load

### ISSUE THREAD

**Issue by aj2duncan** [markheckmann/OpenRepGrid.ic#4]
Hi @markheckmann, 

From the joss review openjournals/joss-reviews#3292 I have installed both the CRAN and github versions of the package but the shiny app won't load. The error message reported is:

```
Warning: Error in rightSidebar: could not find function "rightSidebar"
  [No stack trace available]
```

Thanks.

**Comment by markheckmann** [comment]
@aj2duncan : Thanks, I will have a look at it. Could you add info which OS and which R version you use? Best would be a copy of the sessionInfo() output. Thanks!

**Comment by aj2duncan** [comment]
Ah. Sorry, should have included that!  Before running the app but with the package loaded.

```r
sessionInfo()
```

```
R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.1252  LC_CTYPE=English_United Kingdom.1252   
[3] LC_MONETARY=English_United Kingdom.1252 LC_NUMERIC=C                           
[5] LC_TIME=English_United Kingdom.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] OpenRepGrid.ic_0.5.0

loaded via a namespace (and not attached):
 [1] igraph_1.2.6     Rcpp_1.0.6       magrittr_2.0.1   tidyselect_1.1.1
 [5] R6_2.5.0         rlang_0.4.11     fansi_0.5.0      stringr_1.4.0   
 [9] plyr_1.8.6       dplyr_1.0.6      tools_4.1.0      utf8_1.2.1      
[13] DBI_1.1.1        withr_2.4.2      ellipsis_0.3.2   assertthat_0.2.1
[17] tibble_3.1.2     lifecycle_1.0.0  crayon_1.4.1     zip_2.2.0       
[21] purrr_0.3.4      reshape2_1.4.4   vctrs_0.3.8      glue_1.4.2      
[25] openxlsx_4.2.3   stringi_1.6.1    compiler_4.1.0   pillar_1.6.1    
[29] generics_0.1.0   pkgconfig_2.0.3 
```

After running the app 

```r
sessionInfo()
```

```
R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.1252  LC_CTYPE=English_United Kingdom.1252   
[3] LC_MONETARY=English_United Kingdom.1252 LC_NUMERIC=C                           
[5] LC_TIME=English_United Kingdom.1252    

attached base packages:
[1] splines   stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] igraph_1.2.6             glue_1.4.2               forcats_0.5.1           
 [4] dplyr_1.0.6              purrr_0.3.4              readr_1.4.0             
 [7] tidyr_1.1.3              tibble_3.1.2             ggplot2_3.3.3           
[10] tidyverse_1.3.1          scales_1.1.1             testthat_3.0.2          
[13] reshape2_1.4.4           magrittr_2.0.1           stringr_1.4.0           
[16] DT_0.18                  openxlsx_4.2.3           formattable_0.2.1       
[19] data.table_1.14.0        shinyFeedback_0.3.0      rintrojs_0.3.0          
[22] shinycssloaders_1.0.0    shinydashboardPlus_2.0.1 shinydashboard_0.7.1    
[25] shinyWidgets_0.6.0       shinythemes_1.2.0        shinyjs_2.0.0           
[28] shiny_1.6.0              OpenRepGrid.ic_0.5.0    

loaded via a namespace (and not attached):
 [1] httr_1.4.2        sass_0.4.0        jsonlite_1.7.2    modelr_0.1.8     
 [5] bslib_0.2.5.1     assertthat_0.2.1  cellranger_1.1.0  pillar_1.6.1     
 [9] backports_1.2.1   digest_0.6.27     promises_1.2.0.1  rvest_1.0.0      
[13] colorspace_2.0-1  htmltools_0.5.1.1 httpuv_1.6.1      plyr_1.8.6       
[17] pkgconfig_2.0.3   broom_0.7.6       haven_2.4.1       xtable_1.8-4     
[21] later_1.2.0       generics_0.1.0    ellipsis_0.3.2    cachem_1.0.5     
[25] withr_2.4.2       cli_2.5.0         readxl_1.3.1      crayon_1.4.1     
[29] mime_0.10         evaluate_0.14     fs_1.5.0          fansi_0.5.0      
[33] xml2_1.3.2        tools_4.1.0       hms_1.1.0         lifecycle_1.0.0  
[37] reprex_2.0.0      munsell_0.5.0     zip_2.2.0         compiler_4.1.0   
[41] jquerylib_0.1.4   rlang_0.4.11      grid_4.1.0        rstudioapi_0.13  
[45] htmlwidgets_1.5.3 rmarkdown_2.8     gtable_0.3.0      DBI_1.1.1        
[49] R6_2.5.0          lubridate_1.7.10  knitr_1.33        fastmap_1.1.0    
[53] utf8_1.2.1        stringi_1.6.1     Rcpp_1.0.6        vctrs_0.3.8      
[57] dbplyr_2.1.1      tidyselect_1.1.1  xfun_0.23        
```

**Comment by aj2duncan** [comment]
Hi @markheckmann, 

Just wondering if you've had a chance to think about a timeline for this yet? We can't really move  openjournals/joss-reviews#3292 too far forward until this is resolved. All the best.

**Comment by markheckmann** [comment]
Hi @aj2duncan: The `shinyDashboardPlus` package introduced significant breaking changes with version 2.0.0. This caused the issue. I fixed this now. Sorry for having been a bit slow to react! Thanks!  PS. The fixed version 0.5.1 will probably arrive on CRAN within the next few days.
